### PR TITLE
User otp

### PR DESCRIPTION
**Pull request by valentinludu**

### CODE REVIEW

File: src/prisma/schema.prisma [hunk 1]
```diff
@@ -1994,3 +1995,13 @@ model FormFieldAnswer {
 
   @@index([fieldId])
 }
+
+model UserOTP {
+  id           String  @id @default(uuid()) @db.Uuid
+  secret       String
+  recoveryCode String?
```
Comment (mattcasey) [inline src/prisma/schema.prisma:2002]: make this a separate table including createdAt and usedAt?

Comment (mattcasey) [reply src/prisma/schema.prisma:2002]: add createdAt field on here, activatedAt field

Comment (valentinludu) [reply src/prisma/schema.prisma]: Done

**Review by mattcasey — APPROVED** [review]

File: src/prisma/schema.prisma (hunk 1 above)
Comment (valentinludu) [reply src/prisma/schema.prisma]: Added deletedAt

File: src/prisma/schema.prisma [hunk 2]
```diff
@@ -666,6 +657,7 @@ model User {
   blacklistedUser               BlacklistedUser[]
   proposalEvaluationPermissions ProposalEvaluationPermission[]
   decidedProposalEvaluations    ProposalEvaluation[]
+  userOTP                       UserOTP?
```
Comment (mattcasey) [inline src/prisma/schema.prisma:660]: i would just call this field "otp" everything tied to this user is already user-related

Comment (valentinludu) [reply src/prisma/schema.prisma:660]: Done

**Pull request merged**